go/tools/wtool/main.go

```diff
@@ -0,0 +1,144 @@
+/*wtool augments your bazel WORKSPACE file with new_go_repository entries
+
+Example Usage: wtool com_github_golang_glog com_google_cloud_go
+will add 2 new_go_repository to your WORKSPACE
+by converting com_github_golang_glog -> github.com/golang/glog
+and so forth and then doing a 'git ls-remote' to get
+the latest commit.
+
+if wtool cannot figure out the bazel -> Go mapping, try
+Other Usage: wtool -asis github.com/golang/glog
```

Comment:
The way I use it, I add some dependency in my Go code, say on google.golang.org/genproto/protobuf

I run gazelle, then build and bazel dies saying
.../BUILD:3:1: no such package '@org_golang_google_genproto//protobuf': error loading package 'external': The repository named 'org_golang_google_genproto' could not be resolved and referenced by ...

So I now just type:
 wtool org_golang_google_genproto

Without having to reverse engineer the importpath myself.